src/Microsoft.Data.Analysis/DataFrame.cs

```diff
@@ -39,7 +39,13 @@ public DataFrame()
             _rowCollection = new DataFrameRowCollection(this);
         }
 
-        public DataFrame(IList<DataFrameColumn> columns)
+        public DataFrame(IEnumerable<DataFrameColumn> columns)
+        {
+            _columnCollection = new DataFrameColumnCollection(columns, OnColumnsChanged);
+            _rowCollection = new DataFrameRowCollection(this);
+        }
+
+        public DataFrame(params DataFrameColumn[] columns)
```

Comment:
Do we still need the empty constructor now that we have a `params` overload?

Reply:
We don't. Deleting it.